test-pnp.sh

```diff
@@ -5,31 +5,29 @@ declare -a testCases=(
   "with-mdx"
   # Tests babel config
   "with-styled-components"
+  "with-styled-jsx"
 )
 
 set -e
 
 # Speeds up testing locally
 export CI=1
 
-rm -rf ./e2e-tests
-
-initialDir=$(pwd)
+nextDir=$(pwd)
 
 for testCase in "${testCases[@]}"
 do
-  cd $initialDir
+  testTarget=$(mktemp -d)
```

Comment:
```suggestion
  testTarget=$(mktemp -d next-e2e-XXXX)
```

Makes them easier to clean up

Reply:
@merceyz this creates the temp folder in the current directory unfortunately. I think you want `mktemp -d -t next-e2e` instead, but the `-t` option seems deprecated in newer versions, so I'm not sure if it should be used.

I have a slightly improved version that creates the temp dir only once and then creates subfolders for each test case instead. Also it has a `trap` to remove the temp dir on exit. Let me know if you want me to push that.

```diff
+tempDir=$(mktemp -d)
+trap 'rm -rf -- "$tempDir"' EXIT

 for testCase in "${testCases[@]}"
 do
-  testTarget=$(mktemp -d)
+  testTarget="$tempDir/$testCase"
+
+  mkdir -p "$testTarget"
```